### ISSUE TITLE
Image Description Rendering issue

### ISSUE THREAD

**Issue by a2975667** [arXiv/html_feedback#3439]
### Description

Image /Description requested by the ACM and the copyright information (i.e., CC-NC) for the newer acmart will break the position of these text and the readability.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

fc6330eb-f49f-49ef-aa69-a787076f7fe1

### Paper URL

https://arxiv.org/html/2503.04114v2

### Browser

Chrome/135.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

Feel free to open new tickets for any remaining flaws with the rendering, I will close this one to simplify our bookkeeping.
